app/views/landing_page/show.html.erb

```diff
@@ -1,7 +1,13 @@
 
 <div class="landing-page">
   <% @content_item.blocks.each do |block| %>
-    <%= render "landing_page/blocks/#{block.type}", block: %>
+    <% if block.full_width? %>
+      <%= render "landing_page/blocks/#{block.type}", block: %>
+    <% else %>
+      <div class="govuk-width-container">
```

Comment:
Minor, and not a blocker, but as the thing being rendered is the same, and it's only and extra style being applied, this check could probably inline.

Reply:
Yeah, I was trying to avoid the extra wrapping div if it's not needed, but actually it's probably helpful for it to always be there. Addressed in 631c1d54e1eebd15c295f57ef37e9acdf8fb0329.